i18n/translations.fi.json

```diff
@@ -1597,6 +1598,29 @@
     }
   },
 
+  "details_page": {
+    "site_title": "Details (Lisätieto) | Suomi.fi Design System",
+    "heading": "Details (Lisätieto)",
+    "ingress": "Lisätietokomponentilla voidaan esittää ja piilottaa tiettyyn sivun osaan liittyviä lisätietoja, joita kaikki käyttäjät eivät tarvitse. ",
+    "accessibility_list": {
+      "point_1": "Käytä lisätietokomponenttia ainoastaan sellaisen informaation esittämiseen ja piilottamiseen, joka ei ole kaikille käyttäjille välttämätöntä.",
```

Comment:
Kieliopillisesti ehkä parempaa suomea näin: "Käytä lisätietokomponenttia esittämään ja piilottamaan ainoastaan sellaista informaatiota, joka ei ole välttämätöntä kaikille käyttäjille"